components/Cards/Cards.tsx

```diff
@@ -1,8 +1,7 @@
 import { useRouter } from "next/router";
-import React from "react";
 import { data } from "../../database/data";
 import { IData } from "../../types";
-import { LinkContainer } from "../LinksContainer/LinkContainer";
+import LinkContainer from "../LinkContainer/LinkContainer";
```

Comment:
any specific reason you changed the component export into default export and also deleted index.ts? 

our idea of adding `index.ts` file is to optimise the importing i.e `../LinkContainer` instead of `../LinkContainer/LinkContainer`

Reply:
Removing `index.ts` was just a personal preference. I saw components with index and others without it so I decided on removes it because files tree its more clear. Aso changed the naming to PascalCase to keep them more consistent.
![Refactor](https://i.imgur.com/5Qny3eV.png)
If you want to get back `index.ts` it is ok, tell me and I'll adjust it.
Also if you care of optimizing imports, maybe you want to take a look at [typescript paths](https://www.typescriptlang.org/tsconfig#paths)